cpp/include/raft/neighbors/nn_descent_types.hpp

```diff
@@ -133,6 +148,13 @@ struct index : ann::index {
     return graph_view_;
   }
 
+  [[nodiscard]] inline auto distances() noexcept -> host_matrix_view<DistData_t, int64_t, row_major>
+  {
+    assert(return_distances_);
+    assert(distances_view_.has_value());
+    return distances_view_.value();
```

Comment:
It seems like the test is failing because this part is not using `value_or` to refer to `distances_view_`.
However, I'm not sure how I should use `value_or` here since there is no default value to return in this case (also cannot return `nullptr`).
Any help would be appreciated : )!

Reply:
You could default initialize the member below and that should fix this:
```std::optional<raft::host_matrix_view<DistData_t, int64_t, row_major>> distances_view_; = std::nullopt```

Reply:
Thank you! ☺️